src/OrbitGl/App.h

```diff
@@ -130,6 +130,16 @@ class OrbitApp final : public DataViewFactory,
     return *capture_data_;
   }
 
+  [[nodiscard]] const orbit_client_data::ModuleManager* GetModuleManager() const override {
```

Comment:
Why not return a const ref? `GetCaptureData` returns a const reference for example.

Reply:
As discussed offline. It is stored in the tracks, so we keep it as a pointer.